tests/test_client/tests.py

```diff
@@ -549,6 +550,21 @@ def test_force_login_without_backend(self):
         self.assertEqual(response.context['user'].username, 'testclient')
         self.assertEqual(self.u1.backend, 'django.contrib.auth.backends.ModelBackend')
 
+    @override_settings(AUTHENTICATION_BACKENDS=[
+        'test_client.auth_backends.BackendWithoutGetUserMethod',
+        'django.contrib.auth.backends.ModelBackend'
```

Comment:
Our convention is to include a trailing comma in places like this so if more items to the list later, we don't have to modify this line again.